usr/share/rear/verify/CDM/default/450_start_cdm_rbs.sh

```diff
@@ -0,0 +1,17 @@
+# 450_start_cdm_rbs.sh
+# Start the Rubrik (CDM) RBS Agent
+
+RBA_DIR=/etc/rubrik
+RBA_BIN_DIR=/usr/bin/rubrik
+
+BOOTSTRAP_DAEMON_OPTS="`cat ${RBA_DIR}/conf/bootstrap_flags.conf`"
+AGENT_DAEMON_OPTS="`cat ${RBA_DIR}/conf/agent_flags.conf`"
```

Comment:
@DamaniN 
please no backticks for bash command substitution
because they are so small and a bit hard to see
and openig and closing caracter is same so it cannot be nicely nested.
We perfer `$( command )` with spaces to aid readability.
See "Code must be easy to read" in
https://github.com/rear/rear/wiki/Coding-Style

Use simple `$VARIABLE` when possibe,
see "Variables" in
https://github.com/rear/rear/wiki/Coding-Style

Reply:
Replaced backticks with `$( command )`
Replaced `cat` with `<`